src/app/[locale]/dashboard/_components/products/all-products-table.tsx

```
@@ -0,0 +1,92 @@
+import { toast } from "sonner";
+
+interface Props {
+  products: Product[];
+}
+
+export default function ProductsTable({ products }: Props) {
+
+  // Translations
+  const t = useTranslations("products-table");
+
+  const [items, setItems] = useState(products);
```

ESLint

[error] 'items' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)